README.md

```diff
@@ -80,16 +80,18 @@ Adapt the instructions to suit your cmake generator and operating system.
 Build and install the KDDockWidgets framework:
 
 ```
-$ cmake -G Ninja -DCMAKE_INSTALL_PREFIX=/path/where/to/install ../path/to/kddockwidgets
-$ make
-$ make install
+$ mkdir build && cd $_
+$ cmake -G Ninja -DCMAKE_INSTALL_PREFIX=/kddockwidgets/install/location ..
+$ ninja
+$ ninja install (as root)
```

Comment:
no root needed, it's even discouraged. Only your distro packages should be installed in system directories